stellar/gmp.js

```diff
@@ -64,10 +71,12 @@ if (require.main === module) {
     program.name('gmp').description('Example of Stellar gmp commands');
 
     program
-        .command('send <destinationChain> <destinationAddress> <payload> <gasTokenAddress> <gasFeeAmount>')
+        .command('send <destinationChain> <destinationAddress> <payload>')
         .description('Send gmp contract call')
-        .action((destinationChain, destinationAddress, payload, gasTokenAddress, gasFeeAmount, options) => {
-            mainProcessor(send, [destinationChain, destinationAddress, payload, gasTokenAddress, gasFeeAmount], options);
+        .addOption(new Option('--gas-token-address <gasTokenAddress>', 'gas token address (default: XLM)'))
+        .addOption(new Option('--gas-fee-amount <gasFeeAmount>', 'gas fee amount').default(0))
```

Comment:
```suggestion
        .addOption(new Option('--gas-amount <gasAmount>', 'gas amount').default(0))
```
gas and fee are the same

Reply:
updated to `gasAmount` everywhere